packages/grid_client/tests/modules/applications/taiga.test.ts

```diff
@@ -168,36 +170,36 @@ test("TC2691 - Applications: Deploy Taiga", async () => {
   expect(result[0].mounts[0]["mountPoint"]).toBe(mountPoint);
   expect(result[0].mounts[0]["state"]).toBe("ok");
 
-  const backends = ["http://[" + result[0].planetary + "]:9000"];
-  log(backends);
+  const wgnet = result[0].interfaces[0];
 
   //Name Gateway Model
-  const gw: GatewayNameModel = {
-    name: name,
+  const gateway: GatewayNameModel = {
+    name: subdomain,
+    network: wgnet.network,
     node_id: GatewayNode.nodeId,
-    tls_passthrough: tlsPassthrough,
-    backends: backends,
+    tls_passthrough: false,
```

Comment:
we already have the value of `const tlsPassthrough = false;` declared before we should use it